app/src/main/java/com/kevinlu/watstats/LoginActivity.java

```diff
@@ -0,0 +1,187 @@
+package com.kevinlu.watstats;
+
+import android.app.ProgressDialog;
+import android.os.Bundle;
+import android.util.Log;
+
+import android.content.Intent;
+import android.view.View;
+import android.webkit.WebResourceError;
+import android.webkit.WebResourceRequest;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import butterknife.ButterKnife;
+import butterknife.BindView;
+
+public class LoginActivity extends AppCompatActivity {
+    private static final String TAG = "LoginActivity";
+    private static final int REQUEST_SIGNUP = 0;
+    private WebView webDriver;
+
+    @BindView(R.id.input_watcard)
+    EditText _watcardText;
+    @BindView(R.id.input_password)
+    EditText _passwordText;
+    @BindView(R.id.btn_login)
+    Button _loginButton;
+    @BindView(R.id.about)
+    TextView _aboutLink;
+    @BindView(R.id.privacy_policy)
+    TextView _privacyLink;
```

Comment:
Are these being used somewhere else? If not, make them private.

Underscores in identifiers are generally considered poor style in Java and not used.